src/Main.js

```diff
@@ -0,0 +1,14 @@
+import { BrowserRouter, Routes, Route } from "react-router-dom";
+import App from "./pages/App";
+import FolderPage from "./pages/FolderPage";
+
+export default function Main() {
+  return (
+    <BrowserRouter>
+      <Routes>
+        <Route path="/" element={<App />} />
+        <Route path="/folder" element={<FolderPage />} />
```

Comment:
이런식으로 path는 상수화를 해서 다루는것도 좋아요~
```suggestion
        <Route path={ROUTE_PATHS.HOME} element={<App />} />
        <Route path={ROUTE_PATHS.FOLDER} element={<FolderPage />} />
```